### PR TITLE
Update The-Role-of-Attribute-Stores.md

### PR DESCRIPTION
**Pull request by rotraves-microsoft**
I suggest removing the connection from content referring a product that is not supported anymore. Still, makes sense keeping the description.

Added support to newer versions of SQL and AD;

### CODE REVIEW

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md [hunk 1]
```diff
@@ -17,20 +17,20 @@ For more information about how claims are generated, see [The Role of Claims](Th
 ## How attribute stores fit in with your AD FS deployment goals
 The location of the user attribute store and the location from which users authenticate determine how you design AD FS to support the user identities. Depending on where the attribute store is located and where users will access the application \(in an intranet or on the Internet\), you can use one of the following deployment goals:
 
--   [Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services](/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/dd807071(v=ws.11))—In this goal, users in your organization access an AD FS–secured application or service \(either your own application or service or a partner's application or service\) when the users are logged on to Active Directory in the corporate intranet.
+-   [Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services] — In this goal, users in your organization access an AD FS–secured application or service \(either your own application or service or a partner's application or service\) when the users are logged on to Active Directory in the corporate intranet.
```
Comment (eross-msft) [inline WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:20]: To remove the link is fine, but you need to also remove the brackets around the descriptor text. So it should just look like:  "Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services  — ". If you can make that change to all of these, I'll go ahead and publish.

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md [hunk 2]
```diff
@@ -17,20 +17,20 @@ For more information about how claims are generated, see [The Role of Claims](Th
...
 
 Depending on attribute store placement and other requirements of your organization, you can combine several of these deployment goals to complete the design of your AD FS deployment.
 
 ## Attribute stores that are supported by AD FS
 AD FS supports a wide range of directory and database stores that you can use for extracting administrator\-defined attribute values and populating claims with those values. AD FS supports any of the following directories or databases as attribute stores:
 
--   Active Directory in Windows Server 2003, Active Directory Domain Services \(AD DS\) in Windows Server 2008, AD DS in Windows Server 2012 and 2012 R2, and Windows Server 2016.
+-   Active Directory in Windows Server 2003, Active Directory Domain Services \(AD DS\) in Windows Server 2008, AD DS in Windows Server 2012 and 2012 R2, and Windows Server 2016 and newer;
```
Comment (eross-msft) [inline WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:31]: Please change the semi-colon to a period. This is a singular statement.

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md [hunk 3]
```diff
@@ -17,20 +17,20 @@ For more information about how claims are generated, see [The Role of Claims](Th
...
 
--   All editions of Microsoft SQL Server 2005, SQL Server 2008, SQL Server 2012, SQL Server 2014, and SQL Server 2016
+-   All editions of Microsoft SQL Server 2005, SQL Server 2008, SQL Server 2012, SQL Server 2014, and SQL Server 2016 and newer;
```
Comment (eross-msft) [inline WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:33]: Same comment about the semi-colon to period change.

**Comment by PRMerger10** [comment]
@rotraves-microsoft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by illfated** [comment]
@rotraves-microsoft : Will you be returning to this PR and make the necessary changes, so it can be merged?

**Comment by rotraves-microsoft** [comment]
I was trying to understand how can I do that change without having to submit everything all over again. Do I need to do that to replace the semi-colon by the dot?

**Comment by rotraves-microsoft** [comment]
@illfated

**Comment by illfated** [comment]
Thank you for asking. The main thing to do here, is to go to each of the suggestions from **eross-msft** and find the merge or commit suggestion buttons, then merge the changes to add it to your PR content. That is mostly what is needed.

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md (hunk 1 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:20]: Sorry for not being aware in my previous comment below, I did not notice that the comment from eross-msft is just a comment, and not a change suggestion. Please accept my suggestion for the change with a button to merge/commit the change:
```suggestion
- Provide Your Active Directory Users Access to Your Claims-Aware Applications and Services — In this goal, users in your organization access an AD FS–secured application or service (either your own application or service or a partner's application or service) when the users are logged on to Active Directory in the corporate intranet.
```
Also note: the backslash escape character is usually not needed before regular parenthesis characters.

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md (hunk 2 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:31]: Change suggestion here as well:
```suggestion
- Active Directory in Windows Server 2003, Active Directory Domain Services \(AD DS\) in Windows Server 2008, AD DS in Windows Server 2012 and 2012 R2, and Windows Server 2016 and newer.
```

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md (hunk 3 above)
Comment (illfated) [reply WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:33]:
```suggestion
- All editions of Microsoft SQL Server 2005, SQL Server 2008, SQL Server 2012, SQL Server 2014, and SQL Server 2016 and newer.
```

**Comment by illfated** [comment]
Amended. Added my own suggestions to make it easier for you to add the changes manually, by using the [ Commit suggestion ] buttons.

**Comment by illfated** [comment]
> I was trying to understand how can I do that change without having to submit everything all over again. Do I need to do that to replace the semi-colon by the dot?

GitHub has a feature where you can re-edit the file yourself, but most other users do not have access to editing your PR file except for creating a pull request from their repository to yours (the same file in your PR branch). Feel free to ask, if you want to know more about how to edit your file to add the changes manually in one go.

File: WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md (hunk 3 above)
Comment (billmath) [reply WindowsServerDocs/identity/ad-fs/technical-reference/The-Role-of-Attribute-Stores.md:33]: #sign-off

**Review by billmath — APPROVED** [review]

**Pull request merged**